ui/tests/acceptance/features/hello.feature

```diff
@@ -0,0 +1,8 @@
+#Feature: Hello
```

Comment:
I think this file can go away

Reply:
I wanted to leave the skeleton for Feature Tests so that we avoid the work to re-introduce the infrastructure for them